content/javascript/concepts/dates/terms/setUTCSeconds/setUTCSeconds.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.setUTCSeconds()'
+Description: 'Sets the second value of a date using UTC.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setUTCSeconds()`** method sets the second value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.
+
+## Syntax
+
+```pseudo
+myDate.setUTCSeconds(secondsValue)
+myDate.setUTCSeconds(secondsValue, millisecondsValue)
+```
+
+The `.setUTCSeconds()` method is called on the `myDate` object with an input argument `secondsValue` that passes the new value for seconds.
+
+Optionally, the method can also set the date's millisecond UTC values. The valid number ranges for the time units are as follows:
+
+- **Seconds** (Required): _0_ to _59_
+- **Milliseconds** (optional): _0_ to _999_
+
+## Example
+
+This example sets the second value of the `eventSeconds` object to `35`.
+
+```javascript
```

Comment:
Can we write the result to the console and provide an output ` ```shell ` block showing the output? Like this:

````
This example results in the following output:

```shell
Output here
```
````